hbase/Dockerfile

```diff
@@ -25,14 +25,16 @@ COPY hbase/licenses /licenses
 USER stackable
 WORKDIR /stackable
 
+## copy patches
+COPY --chown=stackable:stackable hbase/stackable/patches /stackable/patches
```

Comment:
Is there a reason to bump this up above the `curl`? This means that the source code will be redownloaded every time the patches are modified.

Reply:
I thought it was more readable since the patches are not only used for hbase patches but phoenix as well, no other reason. Can be changed back as well :)

Reply:
Ideally we always only copy the needed stuff just before we need it to cache as much stuff as possible.
So this means pulling in the hbase and phoenix patches just before build :)

Reply:
Reverted any changes regarding the patches folder and its use in the Dockerfile since the increase of the phoenix version itself removes the need for any phoenix patches for now.